modules/component/src/core/utils/has-zone.ts

```diff
@@ -5,6 +5,9 @@ import { NgZone } from '@angular/core';
  *
  * Determines if the application uses `NgZone` or `NgNoopZone` as ngZone service instance.
  *
+ * It works by detecting if `apply()` is called on a function passed to `runOutsideAngular()`.
+ * The `NgZone` does that while the `NgNoopZone` doesn't do it.
+ *
```

Comment:
I am not quite sure about the meaning of this one, just want to say about this comment, since there is a open PR to add `runOutsideAngular()` to `NgNooZone` too https://github.com/angular/angular/pull/37581/files#diff-9d549aa403494fd043a146b8db64f503R410

Reply:
Hi @JiaLiPassion, thanks for making me aware of this. That comment is outdated. I removed it.

This was my initial implementation which used the different internal implementation of `runOutsideAngular()`: https://github.com/ngrx/platform/pull/2547/commits/89ffe67693c5ddd42a6dcadca52b2f4979bd0261#diff-cf11b5203f02c04c7a9bfeacc85c39c8R20-R27

As far as I understand your PR that would still keep working to detect the `NoopNgZone`, right?

Reply:
So my PR is just make the signature consistent, and sure, this PR will still work!

Reply:
Thanks Jia!